metals/src/main/scala/scala/meta/internal/metals/MetalsExperimental.scala

```diff
@@ -5,7 +5,7 @@ package scala.meta.internal.metals
  * capabilities" in the initialize response.
  */
 
-object MetalsExperimental {
-  // A workaround for https://github.com/microsoft/language-server-protocol/issues/377 until it is resolved
-  val rangeHoverProvider: java.lang.Boolean = true
-}
+case class MetalsExperimental(
```

Comment:
Weird, so was this just an issue with how the object was getting serialized?

Reply:
I checked and yes, Gson is emitting empty json object for scala object
```scala
//> using scala "2.13.8"
//> using lib "com.lihaoyi::pprint:0.7.3"
//> using lib "com.google.code.gson:gson:2.9.0"


import com.google.gson.Gson

object GsonExample extends App {
  val gson = new Gson()

  case object Data {
    final val boolean: java.lang.Boolean = true
  }

  case class Data_2(boolean: Boolean)

  pprint.log(gson.toJson(Data))
  pprint.log(gson.toJson(Data_2(true)))
}
```
```
Gson.scala:17 gson.toJson(Data): "{}"
Gson.scala:18 gson.toJson(Data_2(true)): "{\"boolean\":true}"
```

Reply:
`scala-cli`, providing easy reproductions since 2022. ™️

Reply:
I tested it and it never seemed to have worked with an object. No idea, why it worked before.